dhis-2/dhis-services/dhis-service-program-rule/src/main/java/org/hisp/dhis/programrule/api/RuleEngineEffects.java

```diff
@@ -47,21 +47,21 @@
 
 /**
  * This class holds the action results from rule-engine organized by effect type ({@link
- * ValidationEffect} and {@link NotificationEffect}) and by tracker entity ({@link Enrollment} and
- * {@link Event}
+ * ValidationEffect} and {@link Notification}) and by tracker entity ({@link Enrollment} and {@link
+ * Event}
  */
 @Getter
 public class RuleEngineEffects {
   private final Map<UID, List<ValidationEffect>> enrollmentValidationEffects;
   private final Map<UID, List<ValidationEffect>> eventValidationEffects;
-  private final Map<UID, List<NotificationEffect>> enrollmentNotificationEffects;
-  private final Map<UID, List<NotificationEffect>> eventNotificationEffects;
+  private final Map<UID, List<Notification>> enrollmentNotificationEffects;
```

Comment:
```suggestion
  private final Map<UID, List<Notification>> enrollmentNotifications;
```

and all other fields and methods in this PR 😅

Reply:
Done for notifications ones.
Will tackle validationEffects in next PR(s)